motoman_gp25_support/test/launch_test.xml

```diff
@@ -0,0 +1,59 @@
+
+<!--
+  launch_test.xml - ROSlaunch tests
+
+  Usage Instructions:
+
+  1. Add the following to your CMakeLists.txt:
+    
+    find_package(roslaunch)
+    roslaunch_add_file_check(test/launch_test.xml)
+
+  2. Create a test directory under your package
+  3. Add the "launch_text.xml" file and fill out the test below.  Use the
+     following conventions:
+    a. Encapsulate each launch file test in it's own namespace.  By
+       convention the namespace should have the same name as the launch
+       file (minus ".launch" extension)
+    b. Create tests for each possible combination of parameters.  Utilize
+       sub-namespaces under the main namespace.
+
+  Notes:
+
+  1. XML extension is used in order to avoid beinging included
+  in roslaunch auto-complete.
+
+  2. Group tags with namespaces are used to avoid node name collisions when
+  testing multpile launch files
+-->
```

Comment:
These comments seem unnecessary and could be deleted

Reply:
@gavanderhoorn: Do you agree?

Reply:
I'm OK with this being removed.

The instructions are more for the support package author than for users of the package.